CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 ## main
 
-## 2.23.3
+## 2.24.0
```

Comment:
```suggestion
```

Feel free to ignore, but I thought we were keeping un-released changes under `## main`?